visualization/app/codeCharta/ui/fileChooser/fileChooser.component.spec.ts

```diff
@@ -59,23 +59,11 @@ describe("fileChooserController", () => {
 		})()
 	}
 
-	describe("onImportNewFiles", () => {
-		it("should call $apply", () => {
-			fileChooserController.onImportNewFiles({ files: [] })
-
-			expect($rootScope.$apply).toHaveBeenCalled()
-		})
-
-		it("should not set state if no file loaded", () => {
-			fileChooserController.onImportNewFiles({ files: [] })
-
-			expect(storeSevice.getState().appSettings.isLoadingFile).toBeTruthy()
-		})
```

Comment:
Ideally, a test like this would be kept around. I would just check that `setNewData` is not called in case no file is chosen.